pkg/logs/auditor/auditor.go

```diff
@@ -220,10 +249,26 @@ func (a *Auditor) updateRegistry(identifier string, offset string, tailingMode s
 		// specially want to avoid storing the offset
 		return
 	}
-	a.registry[identifier] = &RegistryEntry{
-		LastUpdated: time.Now().UTC(),
-		Offset:      offset,
-		TailingMode: tailingMode,
+	entry, exists := a.registry[identifier]
+	if !(exists && entry.CurrentConfigID != configID) {
+		a.registry[identifier] = &RegistryEntry{
```

Comment:
Is it expected to not update the registry when an entry exists with a different `CurrentConfigID`?

Reply:
Yes because of #5745 that allows 2 tailers to tail the same file. In that case registry updates from the older tailer should not be accepted.